Project.toml

```diff
@@ -33,6 +34,7 @@ SymEngine = "0.8, 0.9.1"
 SymPy = "1"
 Symbolics = "4, 5"
 julia = "1.6"
+Combinatorics = "1"
```

Comment:
The [compat] section should be ordered alphabetically - this line needs to be moved a bit